config/policies.js

```diff
@@ -213,6 +213,9 @@ module.exports.policies = {
   // Swagger (API doc)
   'v1/swagger/get-yaml': true,
 
+  // Historisation 
```

Comment:
`History` instead of `Historisation` : use the resource name (see my other comment about english translation of french words "historique" and  "historisation")

Reply:
I agree vocabulary need to be clear from the start ;)

Reply:
I'm thinking about something else! We will have a problem when speaking about the history (subject of this PR) and a single history from the cave histories : 

- the first one is about the **data** history (cave.history => a collection of data modifications)
- the 2nd one is about the **cave** history (cave.histories which contains multiple history => a collection of "histoires" about the cave)

So do we keep "history" or do we need to use "dataHistory" instead for example ?

Reply:
With @CecileDestIG, we were thinking about it. What do you think about the name `versionning` or something more related to the idea of version (Because we talk about different version of an entrance for instance) ? 
With that, we don't have problem with history and the distinction is clearer in our opinion.

Reply:
I don't see any problems with `versioning`, it's ok for me 🙂 

I'm thinking to something else in the same "spirit" : `snapshot`. When you retrieve a past version of something, it's something frozen in the past, which will never change. So `snapshot` can be relevant too.

Juste to be precise (once again 😅): be careful between the name `version` and the process `versioning`. I think you will only need the first one for all this PR (i.e `version(s)`).

Reply:
Snapshot seems to be clearer, we will use it.

Reply:
@urien we suggest to use `snapshot(s)` instead of `version(s)` as suggested @Clm-Roig, is it ok ?

Reply:
Oui

Reply:
The "_h" SQL table will not be aligned with the new naming convention but I also prefer "version" and "snapshot" than history ;)